### PR TITLE
Update MP OpenAPI 3.0 API and TCK to RC5

### PR DESCRIPTION
**Pull request by Azquelt**

### CODE REVIEW

File: dev/io.openliberty.org.eclipse.microprofile/openapi.3.0.bnd [hunk 1]
```diff
@@ -18,6 +18,6 @@ Export-Package: \
   org.eclipse.microprofile.openapi.*
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;3.0}
+  @${repo;org.eclipse.microprofile.openapi:microprofile-openapi-api;[3.0,3.1)}
```
Comment (Azquelt) [inline dev/io.openliberty.org.eclipse.microprofile/openapi.3.0.bnd:21]: This change ensures we don't automatically use any future minor (or major) version if it were to be added to oss_dependencies.maven.

**Comment by Azquelt** [comment]
#build

**Review by tevans78 — APPROVED** [review]

**Review by Emily-Jiang — APPROVED** [review]

**Comment by Azquelt** [comment]
#build

**Comment by Azquelt** [comment]
#build

**Pull request merged**